roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -93,6 +93,16 @@
       --selinux-relabel
       --firstboot-command "growpart /dev/sda {{ _root_part }}"
       --firstboot-command "xfs_growfs /"
+      {% if item.value.ip_address is defined %}
```

Comment:
hmmm maybe we should use the "script" capacity here instead of the command one?
```
 --firstboot SCRIPT (see "customize" below)
           Install "SCRIPT" inside the guest, so that when the guest first boots up, the script
           runs (as root, late in the boot process).
```
Advantage here would be to get that script generated on the hypervisor, once per VM. This would be a nice artifact to collect for debugging if needed?

More options: https://manpages.ubuntu.com/manpages/xenial/man1/virt-sysprep.1.html#options

Reply:
Thank you. Let me attempt to use `--firstboot`.